src/develop.js

```diff
@@ -0,0 +1,4 @@
+let a = ("I hope i made it true");
+{
+console.log (a);
+}
```

Comment:
nit: We do not need extra "{}" to set a scope. As we use global var a.
```suggestion
console.log (a);
```

Reply:
ok. I'll do better